include/seqan/file/string_external.h

```diff
@@ -1146,7 +1146,7 @@ namespace seqan
  * @signature String::String(file);
  * @signature String::String(fileName[, openMode]);
  *
- * @param[in]     file     The @link File @endlink to use for reading and writing.  You must ensture that
+ * @param[in]     file     The @link File @endlink to use for reading and writing.  You must ensure that
```

Comment:
actually Double white spaces were not from my branch. They exist also in the master branch.